examples/scenes/src/test_scenes.rs

```diff
@@ -169,10 +169,10 @@ mod impls {
         use PathEl::*;
         move |scene, params| {
             let colors = [
-                Color::from_rgba8(140, 181, 236, 255),
-                Color::from_rgba8(246, 236, 202, 255),
-                Color::from_rgba8(201, 147, 206, 255),
-                Color::from_rgba8(150, 195, 160, 255),
+                palette::css::SKY_BLUE,
+                palette::css::LIGHT_SALMON,
+                palette::css::LAVENDER,
+                palette::css::PALE_GREEN,
```

Comment:
How did you come to these colours? Given that this was originally the same set of colours as at line 349, it's surprising that three of the four colour names used are different.